### PR TITLE
fix: pin dependencies

### PR DESCRIPTION
**Pull request by Lightning00Blade**

### CODE REVIEW

**Review by mathiasbynens — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -28,13 +28,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
```
Comment (thiagowfx) [inline .github/workflows/build.yml:31]: What's the advantage of using hash over major version?

Comment (thiagowfx) [reply .github/workflows/build.yml:31]: @Lightning00Blade

Comment (Lightning00Blade) [reply .github/workflows/build.yml:31]: You can check out this issue https://github.com/puppeteer/puppeteer/issues/10264 in short it's a security thing.

**Review by Vichugo1995 — COMMENTED** [review]
Cómo acceder al check

**Pull request merged**